dataset_doi.yaml

```diff
@@ -27,7 +27,7 @@ epacamd_eia:
   sandbox_doi: 10.5072/zenodo.1072000
 epacems:
   production_doi: 10.5281/zenodo.4127054
-  sandbox_doi: 10.5072/zenodo.672962
+  sandbox_doi: 10.5072/zenodo.1228518
```

Comment:
Why does the concept DOI have to change? I would imagine that we want to keep any new CEMS archives part of the same versioned lineage.

Reply:
😅 that makes a lot of sense!